com.unity.render-pipelines.core/Editor/ProjectorEditor.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace UnityEditor.Rendering
+{
+    [CustomEditorForRenderPipeline(typeof(Projector), typeof(RenderPipelineAsset))]
+    [CanEditMultipleObjects]
+    class ProjectorEditor : Editor
+    {
+        static readonly GUIContent k_Message = EditorGUIUtility.TrTextContent("The active render pipeline does not support the Projector component. If using HDRP, use the Decal Projector component instead.");
```

Comment:
Nitpicking, could be improved for follow up PR, but you could check which RP is running at runtime if it's HDRP, URP, or something else and then display the approapriate message.